src/modules/musig/keyagg_impl.h

```diff
@@ -277,7 +277,7 @@ int secp256k1_musig_pubkey_tweak_add(const secp256k1_context* ctx, secp256k1_pub
     if (overflow) {
         return 0;
     }
-    if (secp256k1_extrakeys_ge_even_y(&cache_i.pk)) {
+    if (xonly && secp256k1_extrakeys_ge_even_y(&cache_i.pk)) {
```

Comment:
To ease maintainability we try to avoid changing files that we didn't add in secp-zkp. `secp256k1_extrakeys_ge_even_y` is a function we get from upstream, so we should change it upstream.